reference/configuration/security.rst

```diff
@@ -319,6 +319,19 @@ Redirecting after Login
 
 .. _reference-security-pbkdf2:
 
+Logout Configuration
+--------------------
+
+invalidate_session
+..................
+
+**type**: ``boolean`` **default**: ``true``
+
+By default, the session of the user is invalidated after the log out process.
+This means that the user will be logged out for all the firewalls defined in
+your application. Set this option to ``false`` to only log out from the current
+firewall.
```

Comment:
maybe some note has to be added that this only applies to the current firewall the user logouts from. So if you always want a user not be logged out in any other firewall, you have to set this to false for *every* firewall defined.